doc/source/dev/roadmap-detailed.rst

```diff
@@ -413,12 +413,12 @@ may also be included in SciPy, especially if no other widely used and
 well-supported package covers the topic.  Also note that *some* duplication
 with downstream projects is inevitable and not necessarily a bad thing.)
 
-The following improvements will help SciPy better serve this role.
+In addition to the items described in the :ref:`scipy-roadmap`, the following
+improvements will help SciPy better serve this role.
 
 - Add fundamental and widely used hypothesis tests:
 
-  - Tukey-Kramer test
-  - Dunnett's test
+  - Dunnett's test and the Student–Newman–Keuls test
```

Comment:
Tukey-Kramer is now in stats as `tukey_hsd`.
Student–Newman–Keuls is another multiple-comparison test that makes use of the studentized range distribution.

Reply:
Imho these kinds of things are too detailed to put on the roadmap. It's a regular (and very niche) feature request, which I'd prefer to not have to think about before there's an actual proposal.

The other changes in this PR are much broader and make more sense. I'd prefer to leave this alone or trim it down - even Dunnett's test doesn't belong here.

Reply:
Well, if I hadn't read it in here, I wouldn't have implemented it this weekend... I need it for work, but knowing that it was here, I wrote it following `scipy.stats` conventions and constraints (i.e. _sans_ pandas).

I do take your point, though. Would contextualizing this list as a set of examples to give an idea of the scope (e.g. "widely used hypothesis tests such as:") fit better?

Reply:
Nice coincidence:)

> Would contextualizing this list as a set of examples to give an idea of the scope (e.g. "widely used hypothesis tests such as:") fit better?

Yes, that'd be great.

Reply:
My intent here was multi-comparison / post hoc tests, in particular.

> widely used post hoc tests such as

@rkern you wrote Student–Newman–Keuls using `studentized_range`? (Dunnet's needs a new distribution, right?)

Reply:
If you add SNK, then add a warning in the docstring.
I'm currently not planning to add it to statsmodels. (and I will not add Fisher's LSD for sure)
https://github.com/statsmodels/statsmodels/issues/8168#issuecomment-1079765074

Reply:
What warning did you have in mind?

> Note: statsmodels does not plan to add this hypothesis test

?

Reply:
I added this to the list because it is used. I suppose it comes down to the question of whether our scope is prescriptive or descriptive.

Reply:
OK, I think I've taken all this into account. I removed explicit mention of Student-Newman-Keuls.